student.rb

```diff
@@ -0,0 +1,12 @@
+require_relative 'person'
+
+class Student < Person
+  def initialize(classroom)
+    super
+    @classroom = classroom
+  end
+
+  def play_hooky
+    "¯\(ツ)/¯"
+  end
+end
```

Comment:
- [ ] Because student class inherits from person.. constructor should extend parent's constructor by adding `@classroom` and a parameter for it

Reply:
Done